docs/source/examples/client-server-request.py

```diff
@@ -0,0 +1,38 @@
+import subprocess
+import logging
+from conu import DockerBackend, DockerRunBuilder
+
+environment = ["-e", "POSTGRESQL_USER=user",
+               "-e", "POSTGRESQL_PASSWORD=pass",
+               "-e", "POSTGRESQL_DATABASE=db"]
+
+backend = DockerBackend(logging_level=logging.DEBUG)
+image = backend.ImageClass('centos/postgresql-96-centos7')
```

Comment:
I hope their image will never break :D It's definitely good to have such a brave test.